content/module-reference/utility-modules/shared/filmstrip.md

```diff
@@ -6,28 +6,24 @@ tags:
 view: darkroom, tethering, map, print
 ---
 
-The filmstrip can be used to quickly switch between images. The images shown are the same as those displayed in the lighttable view and are defined by the currently-selected collection.
+The filmstrip panel allows users to quickly switch between images in the currently-selected collection without leaving the current view. It sits under the centre panel in all views except lighttable where it is replaced by the timeline panel. It can be switched on and off using the shortcut Ctrl+F.
```

Comment:
Haven't we standardized on American spelling?  In that case, it should be "center" not "centre".